_inc/client/at-a-glance/index.jsx

```diff
@@ -53,6 +56,7 @@ class AtAGlance extends Component {
 			siteAdminUrl: this.props.siteAdminUrl,
 			siteRawUrl: this.props.siteRawUrl
 		};
+		const isRewindActive = includes( this.props.activeFeatures, 'jetpack-rewind' );
```

Comment:
This should be updated to use this new constant
https://github.com/Automattic/dops-components/pull/118

Reply:
Addressed by the changes in b5b3ab2e7a4d11004157868e16b73cb31efb0881 for now

Reply:
Just for the records, the feature is still called `jetpack-rewind` so b5b3ab2 was removed